package-lock.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Is this from npm? We use yarn.lock for this, can you remove it from the PR?

Reply:
ya this is not supposed to be with this PR anyway